<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FConsultarfichaempleado</class>
 <widget class="QFrame" name="FConsultarfichaempleado">
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>107</height>
      </size>
     </property>
     <property name="title">
      <string>Resumen de la ficha</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="QLabel" name="foto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>71</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>71</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>81</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../appRes.qrc">:/Images/images/Apu.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>317</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nombre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lbl_nombre">
        <property name="text">
         <string>Apu</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Departamento:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="lbl_seccion">
        <property name="text">
         <string>Asuntos Secretos</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tb_detalle">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../appRes.qrc">
         <normaloff>:/Icons/tango-icons/16x16/actions/document-properties.png</normaloff>:/Icons/tango-icons/16x16/actions/document-properties.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_registrar">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../appRes.qrc">
         <normaloff>:/Icons/tango-icons/16x16/actions/list-add.png</normaloff>:/Icons/tango-icons/16x16/actions/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_baja">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../appRes.qrc">
         <normaloff>:/Icons/tango-icons/16x16/actions/list-remove.png</normaloff>:/Icons/tango-icons/16x16/actions/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_consnomina">
       <property name="text">
        <string>Consultar nómina</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>108</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Buscar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textoBusqueda"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="lista">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>NIF</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="background">
       <color>
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>170</green>
        <blue>127</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true" comment="dfgf" extracomment="sdfggf">Apellidos</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>170</green>
        <blue>255</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Activo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tiempo trabajado en la empresa</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Departamento</string>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>171</green>
        <blue>35</blue>
       </color>
      </property>
     </column>
     <item>
      <property name="text">
       <string>45652349S</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Eduardo</string>
      </property>
      <property name="text">
       <string>Valle Pérez</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="text">
       <string>Sí</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Piloto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>34526718T</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>María Luisa</string>
      </property>
      <property name="background">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="text">
       <string>Jiménez Vera</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="text">
       <string>Sí</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Auxiliar de vuelo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19283456M</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Isabel</string>
      </property>
      <property name="text">
       <string>García Melo</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="text">
       <string>Sí</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>12</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Oficina</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03613576P</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Walterio</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="text">
       <string>Malatesta Prado</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
      <property name="text">
       <string>Sí</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="text">
       <string>Mecánico</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../appRes.qrc"/>
 </resources>
 <connections/>
</ui>
